### PR TITLE
add tercy/rev004

### PR DESCRIPTION
**Pull request by TercyKB**
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/tercy/rev004/info.json
[the remark this answers is not in the document]
Comment (TercyKB) [reply keyboards/tercy/rev004/info.json:1]: Thank you so much for your polite review.
I learned a lot from your suggestions.
I realized that I had not understanded the source code I refered.
Qmk_firmware requires less codes, it is wonderfull!!

I need time to read the PR Checklist and the CONTRIBUTING document.
I will fix readme.md after that.

**Comment by TercyKB** [comment]
I enabled the PCB data to be downloaded for hardware availability, and updated readme.md.
Please review the PR again.
Thank you.

File: keyboards/tercy/rev004/readme.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# Tercy/rev004
+
+![Tercy/rev004](https://etlife.info/wp-content/uploads/2024/05/Tercy_rev004_keyboard.jpg)
```
Comment (drashna) [inline keyboards/tercy/rev004/readme.md:3]: Please rehost these images on Imgur or similar image hosting sit.   Blogs like this are nvere a good long term solution.

Comment (TercyKB) [reply keyboards/tercy/rev004/readme.md]: Thank you for your advice. I rehosted the images to Imgur.

**Review by drashna — APPROVED** [review]

**Comment by TercyKB** [comment]
Is there anything I have to do? just to wait?

**Pull request closed without merging**